AnkiDroid/src/main/java/com/ichi2/anki/MediaRegistration.java

```diff
@@ -76,10 +79,28 @@ public String loadImageIntoCollection(Uri uri) throws IOException {
             fileName = fileNameAndExtension.getKey();
         }
 
-        File clipCopy = File.createTempFile(fileName, fileNameAndExtension.getValue());
-        String tempFilePath = clipCopy.getAbsolutePath();
-        long bytesWritten = CompatHelper.getCompat().copyFile(fd, tempFilePath);
+        File clipCopy;
+        long bytesWritten;
+
+        if (!".png".equals(fileNameAndExtension.getValue()) && CollectionHelper.getInstance().getCol(mContext).getConf().optBoolean("pastePNG")) {
+            clipCopy = File.createTempFile(fileName, ".png");
+            bytesWritten = CompatHelper.getCompat().copyFile(fd, clipCopy.getAbsolutePath());
+            Bitmap bm = BitmapFactory.decodeFile(clipCopy.getAbsolutePath());
+            try (FileOutputStream outStream = new FileOutputStream(clipCopy.getAbsolutePath())) {
+                bm.compress(Bitmap.CompressFormat.PNG, 100, outStream);
+                outStream.flush();
+            } catch (IOException e) {
+                Timber.w("MediaRegistration : Unable to convert file to png format");
+                AnkiDroidApp.sendExceptionReport(e, "Unable to convert file to png format");;
+                UIUtils.showThemedToast(mContext, mContext.getResources().getString(R.string.multimedia_editor_something_wrong_details, e.getMessage()), true);
```

Comment:
sorry - you cannot just commit my suggestion because now this string won't coincide :-) 
```suggestion
                UIUtils.showThemedToast(mContext, mContext.getResources().getString(R.string.multimedia_editor_png_paste_error, e.getMessage()), true);
```

With this one it might? And I may squash merge it